charts/armo-components/templates/armo-collector-deployment.yaml

```diff
@@ -1,5 +1,4 @@
-{{- if .Values.armoCollector.enabled  }}
-{{ template "account_guid" . }}
+{{- if and .Values.armoCollector.enabled .Values.accountGuid }}
```

Comment:
`{{- if .Values.armoCollector.enabled and .Values.accountGuid }}`
Right? am I missing something?

Reply:
see https://helm.sh/docs/chart_template_guide/function_list/
operator operand1 operand2 etc (polish notation)

Reply:
Ok, tnx :)